test/unit/nedap/utils/test/matchers.cljc

```diff
@@ -0,0 +1,20 @@
+(ns unit.nedap.utils.test.matchers
+  (:require
+   #?(:clj [clojure.test :refer [deftest testing are is use-fixtures]] :cljs [cljs.test :refer-macros [deftest testing is are] :refer [use-fixtures]])
```

Comment:
nitpicking, split the `clj` and `cljs` cases to their own lines

Reply:
Hmmm... apparently this is how things are done in matcher-combinators? As I see the same in PR #29...

Is there a recommended/linter/preferred way to deal with formatting these reader conditionals?

Reply:
I think cljfmt could theoretically deal with these, but is not set up afaik.

We tend to do single-line reader conditionals for imports and requires, and multiline otherwise.

Probably originated from the fact that the requires are equal in effect